src/tabs/borderless.scss

```diff
@@ -9,6 +9,11 @@
     padding: 0 ($iui-component-padding-horizontal * 2);
     background-color: transparent;
 
+    &::after {
```

Comment:
Is this part needed?

Reply:
Without that CSS I'm getting this incorrect placement
![Screen Shot 2021-08-17 at 12 51 32 PM](https://user-images.githubusercontent.com/849817/129767898-b2f95e6e-36eb-461a-b831-2cab7915dffd.png)